src/librustdoc/passes/calculate_doc_coverage.rs

```diff
@@ -238,7 +238,7 @@ impl<'a, 'b> fold::DocFolder for CoverageCalculator<'a, 'b> {
                     &i.attrs
                         .doc_strings
                         .iter()
-                        .map(|d| d.doc.as_str())
+                        .map(|d| d.doc.to_string())
```

Comment:
Why did this change? It seems a little silly to allocate a String just to immediately drop it.

Reply:
For this reason:

```
   --> src/librustdoc/passes/calculate_doc_coverage.rs:243:26
    |
243 |                         .join("\n"),
    |                          ^^^^ method not found in `Vec<SymbolStr>`
```

Reply:
I'll simply use the existing methods I guess. :)